### DISCUSSION TITLE
use String-ts  in Valibot to split join etc and so more

### DISCUSSION THREAD

**Discussion by jojojojojoj5564656465465** [open-circle/valibot#959] [Ideas]
[Repo](https://github.com/gustavoguichard/string-ts)

**Comment by fabian-hiller** [comment]
You can already combine `string-ts` with `valibot`. Or do you have a specific idea for extending Valibot's API?

**Comment by jojojojojoj5564656465465** [comment]
I was thinking maybe you could use string ts to extend valibot

Le jeu. 5 déc. 2024, 04:53, Fabian Hiller ***@***.***> a
écrit :

**Reply by fabian-hiller** [reply]
Valibot is dependency free. I don't think we will add `string-ts` to our bundle, but we can provide a better API to use both together if more people are interested.
